pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
     "Natural Language :: English",
     "Topic :: Scientific/Engineering",
 ]
-packages = [{include = "mxcube3", from = "."}]
+packages = [{include = "mxcubeweb", from = "."}]
```

Comment:
This line is probably useless now that the distribution package name matches the name of the top-level import package.

Reply:
Oh yes

Reply:
I see it's removed now, thanks :)